src/app.coffee

```diff
@@ -34,7 +34,7 @@ app.config (
 
   $locationProvider.hashPrefix '!'
 
-  $urlRouterProvider.otherwise '/login/trello'
+  $urlRouterProvider.otherwise '/sprint/current'
```

Comment:
But it works...

On Tue, Jan 5, 2016 at 2:44 PM nic.girault <notifications@github.com> wrote:

> In src/app.coffee
> <https://github.com/theodo/not-so-shitty/pull/47#discussion_r48845389>:
>
> > @@ -34,7 +34,7 @@ app.config (
> >
> >    $locationProvider.hashPrefix '!'
> >
> > -  $urlRouterProvider.otherwise '/login/trello'
> > +  $urlRouterProvider.otherwise '/sprint/current'
>
> this is not ok...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theodo/not-so-shitty/pull/47/files#r48845389>.
>